homeassistant/components/media_player/sonos.py

```diff
@@ -441,6 +443,7 @@ def update(self):
                     support_next_track = False
                     support_stop = False
                     support_pause = False
+                    support_play = False
```

Comment:
This is in pretty bad need of refactoring, but I'd rather not take that on in this PR.